src/gui/export.py

```diff
@@ -59,14 +64,71 @@ def export(self):
             self.progressBar.setFormat(
                 f"Export de la couche {layer.name()}, Total : %p%"
             )
+
+            # Export is done from copy to avoid bug with join field, due to
+            # the table dock.
+            layer = QgsVectorLayer(layer.source(), layer.name())
+
+            # Add Lon/Lat field
+            if layer.geometryType() != 4:
```

Comment:
Can we use an enum instead of the `4` value?

Reply:
:+1: